releasenotes/notes/remove-deprecated-unroller-4693330708c681e0.yaml

```
@@ -0,0 +1,26 @@
+---
+upgrade:
+  - |
+    Removes the deprecated ``Unroller`` class in :mod:`qiskit.transpiler.passes.basis`.
+    This class was deprecated in Qiskit Terra 0.45 and has been replaced by 
+    the combination usage of :class:`.BasisTranslator` and :class:`.UnrollCustomDefinitions`.
+
+    Note that :class:`.BasisTranslator` and :class:`.UnrollCustomDefinitions` take different 
+    arguments than ``Unroller``, as they requires a `EquivalenceLibrary` object to be passed in. 
+
+    Example Usage:
+    Instead of using ``Unroller`` as follows:
+      pm = PassManager([
+          Unroller(basis_gates)
+      ])
+    
+    First create a EquivalenceLibrary like:
+      from qiskit.circuit.library.standard_gates.equivalence_library import (
+          StandardEquivalenceLibrary as std_eqlib,
+      )
+    and then use ``BasisTranslator`` and ``UnrollCustomDefinitions`` as follows:
+      pm = PassManager([
+          UnrollCustomDefinitions(std_eqlib, basis_gates)
+          BasisTranslator(std_eqlib, basis_gates),
+      ])
```

I'm not sure if that would currently render nicely, how about the following?
```suggestion
    Where previously ``Unroller(basis_gates)`` could be used,
    you can now use::
    
      from qiskit.circuit.library.standard_gates.equivalence_library import (
          StandardEquivalenceLibrary as std_eqlib,
      )
      pm = PassManager([
          UnrollCustomDefinitions(std_eqlib, basis_gates)
          BasisTranslator(std_eqlib, basis_gates),
      ])
      translated = pm.run(circuit)
```